src/ui-test/tests/debugger.test.ts

```diff
@@ -65,19 +67,26 @@ describe('Camel Debugger tests', function () {
         driver = VSBrowser.instance.driver;
 
         await VSBrowser.instance.openResources(path.resolve('src', 'ui-test', 'resources'));
+        console.log('src/ui-test/resources opened');
```

Comment:
I am not a fan of this logs because they are always visible directly in logs during tests but not a blocker.. it would be nice to add better logging option for extester to enable debug logs